### PR TITLE
Revisions to housing burden calculation and for addressing standard error measurements using CalEnviroScreen methodology

### PR DESCRIPTION
**Pull request by saran-ahluwalia**
Addresses #1024 and #1029.

There are two notebooks that address two fundamental problems: 1) reliability based on ACS sample estimates of the population and 2) the original request posed by Blair.  

1. One notebook proposes an alternative methodology to address the systematic problem in CHAS data. Specifically, CHAS data is based on American Communities Survey (ACS) estimates, which come from a sample of the population, and hence are unreliable if based on a small sample or population size.

The standard error and relative standard error were used to evaluate the reliability of each estimate using CalEnviroScreen’s [methodology](https://oehha.ca.gov/media/downloads/calenviroscreen/report/ces3report.pdf) (page 129).

This is was done to propose one way to address a comment about small sample size checks (line 274 in etl.py in the `hud_housing` module). The related task can be found [here](https://github.com/usds/justice40-tool/issues/1029).

The sources for this methodology are cited in this notebook and I have added detailed comments there.

I also compared the derived scores with the 4.0 version of Calenvironscreen (notebook was not committed), but I have attached a snapshot of the two derived column estimate, comparing (left to right):

hbrd_score | hbrd_rank  | Housing Burden | Housing Burden Pctl
-- | -- | -- | --

Screen Shot 2021-12-10 at 10 21 41 AM 

**Note**: Although this methodology reflects 3.0, the updated 4.0 still maintains the same [methodology](https://oehha.ca.gov/media/downloads/calenviroscreen/document/calenviroscreen40summaryofchangesf2021.pdf) (please see page 14).

2. The second notebook makes adjustments as a revision to the denominator that more closely resembles the (as I understand) intent behind Blair's response. This actually addresses #1024.

@lucasmbrown-usds @switzersc-usds - Would it make sense to send this back to Blair for his re-review so that we could get some of his thoughts on the proposed methodology and the interpretation of poverty - as he originally shared?

### CODE REVIEW

File: data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read in the data from https://www.huduser.gov/portal/datasets/cp.html\n",
+    "housing = pd.read_csv(\"Table8.csv\", \n",
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb:87]: can you make sure your code works for anyone who wants to run it? 

in this case, just copy from the ETL script to download this table and unzip it. thanks!

File: data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb [hunk 2]
```diff
@@ -0,0 +1,686 @@
...
+    "# )\n",
+    "# divided by\n",
+    "# (\n",
+    "#     Total # of Owner Occupied Units that meet criterion for poverty\n",
+    "#     + Total # of Renter Occupied Units that meet criterion for poverty\n",
+    "#     - # of Owner Occupied Units with HAMFI Not Computed\n",
+    "#     - # of Renter Occupied Units with HAMFI Not Computed\n",
+    "# )\n",
+    "\n",
+    "housing['denominator_post'] = housing[\n",
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb:389]: It looks like something in this math is off? The ratio should be a number between 0 and 1 but yours goes up to 1.13 

Screen Shot 2021-12-10 at 5 49 18 PM

File: data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb (hunk 1 above)
Comment (saran-ahluwalia) [reply data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb:87]: Got it, thanks

File: data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb (hunk 2 above)
Comment (saran-ahluwalia) [reply data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb:389]: @lucasmbrown-usds Thanks for pointing out that the ratios are greater than 1 for 20 census tracts. Some context:

Screen Shot 2021-12-11 at 5 38 42 AM
Screen Shot 2021-12-11 at 5 43 57 AM

I can double-check the legend in CHAS. After that, I did want to run this by our contact in HUD.

File: data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb (hunk 1 above)
Comment (saran-ahluwalia) [reply data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb:149]: @lucasmbrown-usds I have added the ETL as per your request

File: data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb (hunk 2 above)
Comment (saran-ahluwalia) [reply data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb:450]: @lucasmbrown-usds - I was reviewing this issue #283. I would like to discuss this with you at some point.

**Comment by saran-ahluwalia** [comment]
@lucasmbrown-usds @BethMattern Regarding this issue, #244, my questions are:

1. Did we ever come to a consensus on this?
2. After reviewing the analysis in this housing burden PR's outcome results and eligible tracts, I think one reason why such a phenomenon occurs lies in the usage of how relative population estimates are computed using the ranked percentiles.

**Comment by saran-ahluwalia** [comment]
@lucasmbrown-usds @esfoobar-usds I am moving this in order for someone else to take over. This thread and context is sufficient.

**Comment by lucasmbrown-usds** [comment]
Closing based on @saran-ahluwalia 's recommendation that we *not* move to this alternative definition of housing burden.

**Pull request closed without merging**